spec/unit/controllers/runtime/app_bits_upload_controller_spec.rb

```diff
@@ -132,7 +132,7 @@ def make_request
           end
 
           context 'with at least one resource and an application' do
-            let(:req_body) { { resources: Oj.dump([{ 'fn' => 'lol', 'sha1' => 'abc', 'size' => 2048 }]), application: valid_zip } }
+            let(:req_body) { { resources: Oj.dump([{ 'fn' => 'lol', 'sha1' => 'da39a3ee5e6b4b0d3255bfef95601890afd80709', 'size' => 2048 }]), application: valid_zip } }
```

Comment:
**question (non-blocking)**: why do we want to have valid sha in units? Aren't we mocking actual library?